typescript-requiring-type-checking.js

```diff
@@ -15,23 +15,23 @@ module.exports = {
         /*
```

Comment:
Discussion for recommended rule: `'@typescript-eslint/no-implied-eval'`

Reply:
This rule is enabled in the extensions.
**Proposal**
Leave this rule un-configured.

Reply:
We can close this thread with no action